.github/workflows/build.yaml

```diff
@@ -5,19 +5,15 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        ghc: ['8.8.3', '8.10.2', '9.0.1', '9.2.1']
+        ghc: ['9.2.8', '9.4.5', '9.6.2']
         os: [ubuntu-latest, macOS-latest, windows-latest]
-        exclude:
-          # a test suite dependency fails on windows/ghc8.8.3
-          - os: windows-latest
-            ghc: '8.8.3'
       fail-fast: false
     name: Build with GHC ${{ matrix.ghc }} on ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v2
       - name: Setup Haskell
-        uses: haskell/actions/setup@v1
+        uses: haskell/actions/setup@v2.4.3
         with:
           cabal-version: latest
           ghc-version: ${{ matrix.ghc }}
-      - run: cabal v2-test
+      - run: cabal v2-build
```

Comment:
Doesn't this mean the tests are no longer run at all? I mean it's debatable if running the tests should be a separate step, but they should definitely be run at some point.

Reply:
Not sure why I changed this to be build only 🤔 maybe it’d fail calling cabal test on a package that doesn’t have tests.